### PR TITLE
Fix several spelling corrections

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -21304,6 +21305,8 @@ modfiying->modifying
 modfy->modify
 modfying->modifying
 modications->modifications
+modidfied->modified
+modidfy->modify
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:21309]:
```text
modidfication->modification
modidfications->modifications
modidfier->modifier
modidfiers->modifiers
modidfies->modifies
modidfying->modifying
```
* https://github.com/search?q=modidfication&type=code
* https://github.com/search?q=modidfications&type=code
* https://github.com/search?q=modidfier&type=code
* https://github.com/search?q=modidfiers&type=code
* https://github.com/search?q=modidfies&type=code
* https://github.com/search?q=modidfying&type=code

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -27800,6 +27806,8 @@ renegoziator->renegotiator
 renegoziators->renegotiators
 reneweal->renewal
 renewl->renewal
+renforce->reinforce
+renforced->reinforced
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:27810]: `renforce` should be French?
```text
renforcement->reinforcement
renforcements->reinforcements
renforces->reinforces
```
* https://github.com/search?q=renforcement&type=code
* https://github.com/search?q=renforcements&type=code
* https://github.com/search?q=renforces&type=code

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -32615,6 +32623,7 @@ temorarily->temporarily
 temorary->temporary
 tempalte->template
 tempaltes->templates
+temparally->temporarily, temporally,
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:32626]:
```text
temparal->temporal
```
* https://github.com/search?q=temparal&type=code

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -33217,6 +33226,8 @@ Tolkein->Tolkien
 tollerable->tolerable
 tollerance->tolerance
 tollerances->tolerances
+tolorance->tolerance
+tolorant->tolerant
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:33230]:
```text
tolorances->tolerances
```
* https://github.com/search?q=tolorances&type=code

**Review by vikivivi — CHANGES_REQUESTED** [review]

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt:27810]: Yes, but sometimes french words make it in to english source. So i think it's valid

**Comment by luzpaz** [comment]
@vikivivi thanks! Added your mods and rebased. Please re-review. TIA!

**Review by vikivivi — APPROVED** [review]
LGTM

**Pull request merged**